specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1453 @@
+      ],
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "ipTrafficProperties": {
+          "$ref": "#/definitions/IPTrafficProperties"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "sourceResourceId": {
+          "minLength": 1,
+          "type": "string"
+        },
+        "destinationResourceId": {
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "AnalysisRun": {
+      "description": "Instance of analysis result performed given a analysis intent and configuration snapshot.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AnalysisRunProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "AnalysisRunList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AnalysisRun"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "AnalysisRunProperties": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "snapshotId": {
+          "type": "string"
+        },
+        "analysisResult": {
+          "type": "string"
+        },
+        "errorMessage": {
+          "type": "string"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        }
+      }
+    },
+    "ConfigurationSnapshot": {
+      "description": "Instance of network snapshot used to perform the analysis.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ConfigurationSnapshotProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "ConfigurationSnapshotList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigurationSnapshot"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "ConfigurationSnapshotProperties": {
+      "required": [
+        "scopes"
+      ],
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigurationSnapshotScope"
+          },
+          "x-ms-identifiers": []
+        },
+        "createTime": {
```

Comment:
>createTime [](http://example.com/codeflow?start=9&length=10)

This can be removed if you add systemData support. Ref: [https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemData](https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemData).